.github/workflows/plugin-ci.yml

```diff
@@ -22,6 +36,23 @@ on:
         required: false
         type: string
 
+      region:
+        default: us-east-2
```

Comment:
The default region shouldn't be `us-east-1` ?

Reply:
nope this is `us-east-2`. Not sure why we have chosen this.